underlay/src/main/resources/config/datamapping/aouRT/entity/activitySummary/all.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
-    row_id,
```

Comment:
row_id is not a source table field. It is calculated since person_id is not unique